Source/items.cpp

```diff
@@ -780,8 +779,7 @@ void CalcPlrItemVals(int p, BOOL Loadgfx)
 	int lmin = 0; // minimum lightning damage
 	int lmax = 0; // maximum lightning damage
 
-	// didn't find a use for t for now
-	// int t;
+	// didn't find a use for t for now int t;

```

Comment:
Put `// int t;` on a separate line?

Reply:
How about just removing the line?

Reply:
I wonder whether it originates from PSX port.